hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1905,10 +1905,10 @@ public void testPayloadClassUpdateWithCOWTable() throws Exception {
     assertTrue(props.containsKey(HoodieTableConfig.RECORD_MERGE_STRATEGY_ID.key()));
 
     //now create one more deltaStreamer instance and update payload class
```

Comment:
I think going forward we assume that the table is created with the payload class and cannot be updated after the fact since this can cause consistency issues.